frontend/src/hooks/useEltord.ts

```
@@ -0,0 +1,231 @@
+                timestamp: new Date().toISOString(),
+                level: 'INFO',
+                message: 'Relay eltord process deactivated',
+                source: 'system'
+              }))
+            }
+            break
+          case 'eltord-error':
+            console.error('Eltord error:', payload)
+            // Handle specific error cases that might indicate state desync
+            const payloadStr = typeof payload === 'string' ? payload : String(payload)
+            if (payloadStr.includes(`No eltord ${mode} process is currently running`)) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)